style.css

```
@@ -138,3 +138,296 @@ img {
+  letter-spacing: 0.02em;
+  color: var(--white);
+  margin: 0;
+  width: max-content;
+}
+
+.about-title-text-accent-red {
+  color: var(--accent-red);
+}
+
+.about-title-text-accent-yellow {
+  color: var(--accent-yellow);
+}
+
+.about-details {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.about-detail-text {
+  font-family: var(--font-family-body);
+  font-size: 16px;
+  color: var(--white);
+  margin: 0;
+  max-width: 460px;
+}
+
+.about-detail-img {
+  width: 320px;
+  height: 180px;
+  object-fit: cover;
+}
+
+@media (max-width: 750px) {
+  .about {
+    padding: 40px 20px;
+  }
+
+  .about-title-text {
+    font-size: 24px;
+  }
+
+  .about-detail-text {
+    width: 100%;
+    max-width: none;
+  }
+
+  .about-detail-img {
+    display: none;
+  }
+}
+
+/* Courses */
+
+.courses {
+  background-color: var(--background-navy);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 80px 0;
+}
+
+.courses-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 40px;
+  width: 100%;
+  max-width: 800px;
+  gap: 40px;
+}
+
+.courses-title {
+  font-family: var(--font-family-heading);
+  font-size: 32px;
+  letter-spacing: 0.02em;
+  color: var(--white);
+  margin: 0;
+}
+
+.courses-cards {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  width: 100%;
+}
+
+.courses-card {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 150px;
+  filter: drop-shadow(4px 4px 0 var(--accent-red));
+  transition: 0.3s;
+  position: relative;
+  text-decoration: none;
+}
+
+.courses-card:hover {
+  background-color: var(--course-hover-yellow);
+  filter: drop-shadow(12px 12px 0 var(--accent-red));
+  transition: 0.3s;
+}
+
+.courses-card:hover .courses-card-body {
+  background-color: var(--course-hover-yellow);
+  transition: 0.3s;
+}
+
+.courses-card-title {
+  background-image: url(./img/live.png);
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 250px;
+  height: 150px;
+  gap: 10px;
+  position: relative;
+  z-index: 1;
+}
+
+.courses-card-title::after {
+  content: "";
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--course-overlay);
+  position: absolute;
+}
+
+.courses-card-title-text {
+  font-family: var(--font-family-heading);
+  font-size: 32px;
+  font-weight: 400;
+  letter-spacing: 0.05em;
+  color: var(--white);
+  margin: 0;
+  z-index: 1;
+}
+
+.courses-card-body {
+  background-color: var(--accent-yellow);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex: 1;
+  height: 150px;
+  padding: 20px;
+  gap: 10px;
+  transition: 0.3s;
+}
+
+.courses-card-body:last-child {
+  align-items: end;
+}
```

```suggestion
.courses-card-body {
  background-color: var(--accent-yellow);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: end;
  flex: 1;
  height: 150px;
  padding: 20px;
  gap: 10px;
  transition: 0.3s;
}
```
疑似クラスが活用できていてとてもステキです！

ただ今回の場合は `.courses-card-body` は基本的に最後の要素として配置されているので、このように書いてしまうので問題ない気がします！
もしくは、 `courses-card-body-detail` に対して `align-self` を設定するのもありです。